### ISSUE TITLE
Flutter flavors [Target file "lib/main.dart" not found]

### ISSUE THREAD

**Issue by infureal** [Appollo-CLI/Appollo#18]
I have multiple flavors for one applicaton. So I don't have standart main.dart in my project.
```
-lib
-- main_dev.dart
-- main_staging.dart
-- main_production.dart
```

For android I would run `flutter build appbundle -t ./lib/main_production.dart --flavor=production`

But when I run `build start --build-type publication` returns `Target file "lib/main.dart" not found`. I didn't find any options or tutorials where can I specify needed flavor.

**Comment by maxcmoi89** [comment]
@tchavetdeuse is working on this. We planned a release on Thursday if our test go well.

**Comment by tchavetdeuse** [comment]
Hi, this is now fixed in version 1.0.2, you can supply the --target and --flavor parameters or specify them in a .appollo file
